test/lit/passes/cfp-reftest-desc.wast

```diff
@@ -106,3 +106,91 @@
   )
 )
 
+(module
+  (rec
+    ;; CHECK:      (rec
+    ;; CHECK-NEXT:  (type $super (sub (descriptor $super.desc (struct))))
+    (type $super (sub (descriptor $super.desc (struct))))
+    ;; CHECK:       (type $super.desc (sub (describes $super (struct (field (ref (exact $func)))))))
+    (type $super.desc (sub (describes $super (struct (field (ref (exact $func)))))))
```

Comment:
What purpose does the funcref field serve? Can it be removed?

Reply:
Good point, removed.

Reply:
Looks like it's still there. Forgot to push?

Reply:
Oops, yes, pushed now.